content/docs/user-guide/basic-concepts/dependency.md

```diff
@@ -4,5 +4,5 @@ match: [dependency, dependencies]
 ---
 
 A file or directory (possibly tracked by DVC) recorded in the `deps` section of
-a DVC-file (stage file). See `dvc run`. Stages are invalidated when any of their
-dependencies change.
+a `.dvc` file (stage file). See `dvc run`. Stages are invalidated when any of
```

Comment:
It's dvc.yaml here. There are no deps in .dvc files.
```suggestion
a `dvc.yaml` file. See `dvc run`. Stages are invalidated when any of
```
> Please check formatting though

Reply:
I'm assuming you also want all occurrences of `dvc.yaml` file and `.dvc` file to be linked to `/doc/user-guide/dvc-file-format` ?

Reply:
> I'm assuming you also want all occurrences of dvc.yaml file and .dvc file to be linked to /doc/user-guide/dvc-file-format ?

Correct. Well, not all, just the first ones in each section of a doc. No need to repeat links over and over multiple times per paragraph/section so please check whether they may be linked recently before adding missing links.

Reply:
I see this one is fixed in https://github.com/iterative/dvc.org/pull/1408/files#diff-fae5beb83c8343057fae2259866c8746. Thanks!